packages/web/src/app/components/editor/Editor.tsx

```diff
@@ -90,8 +105,10 @@ const Editor = ({
   };
 
   const handleClear = () => {
-    setPixelsHistory([EMPTY]);
-    setPixels(EMPTY);
+    if (confirm('Are you sure you want to erase your drawing?')) {
```

Comment:
Oof, I was just filling in a tile and hit "clear" without thinking and without a confirmation I wiped my progress 🙈